src/main/java/com/stripe/param/SourceUpdateParams.java

```diff
@@ -4,13 +4,18 @@
 
 import com.google.gson.annotations.SerializedName;
 import com.stripe.net.ApiRequestParams;
+import com.stripe.param.common.EmptyParam;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import lombok.Getter;
 
 public class SourceUpdateParams extends ApiRequestParams {
+  /** Amount associated with the source. */
+  @SerializedName("amount")
+  Long amount;
+
```

Comment:
Flagging this should also be in the changelog.

Reply:
👍 Nice catch.